app/views/layouts/listnav/_container_link_to.html.haml

```diff
@@ -0,0 +1,43 @@
+- return if object.nil?
+- if object.kind_of?(ActiveRecord::Relation)
+  -# has_many relation
+  - klass = object.klass.base_model
+  - if klass == Container
+    - feature = 'containers'
+  - else
+    - feature ||= "#{klass.name.underscore}_show_list"
+  - label = ui_lookup(:models => klass.name)
+  - if role_allows(:feature => feature)
+    - count = object.count
+    - if count == 0
+      %li.disabled
+        = link_to(_("#{label} (#{count})"), "#")
+    - elsif count == 1
+      - object = object.first
+      - klass = object.class.base_model
+      %li
+        = link_to(_("#{label} (#{count})"),
+                  {:controller => "#{klass.name.underscore}", :action => 'show', :id => object},
+                  :title => _("View #{label} #{object.name}"))
+    - else
+      %li
+        = link_to(_("#{label} (#{count})"),
```

Comment:
Well, the still-too-short version is: `_("foo")` will look up "foo" in a big (language-specific) hash, to map it to the appropriately translated string. We could obviously do with more documentation on exactly how the nuances work, but even a cursory consideration should show that "don't put a numeric parameter, that could have any value, into the lookup string" is going to be right near the top of the list. (More generally, the first argument to `_()` should almost always be a static string -- otherwise we can't detect it for translation.)

In this particular case, as we've already translated the `label`, we might be able to skip `_` entirely. But otherwise, we'd want something like `_("%s (%d)") % [label, count]`. I'm not sure whether we consider "here are some parens" to be translatable. (@martinpovolny?)